products/electron.md

```diff
@@ -80,7 +75,7 @@ releases:
     releaseDate: 2022-02-01
     eol: true
```

Comment:
```suggestion
    eol: 2022-08-02
```

According to https://www.electronjs.org/docs/latest/tutorial/electron-timelines.